src/components/Tooltip/Tooltip.js

```diff
@@ -216,12 +216,14 @@ class Tooltip extends Component {
     /**
      * The description of the default tooltip icon, to be put in its SVG 'aria-label' and 'alt' .
      */
-    iconDescription: PropTypes.string,
-
-    /**
-     * The title of the default tooltip icon, to be put in its SVG `<title>` element.
-     */
-    iconTitle: PropTypes.string,
+    iconDescription: ({ triggerText, iconDescription }) => {
```

Comment:
Seems close to the use case of: https://github.com/IBM/carbon-components-react/blob/v7.1.0/src/prop-types/isRequiredOneOf.js